### PR TITLE
Updated dialog title to take h4 stylings

### PR DESCRIPTION
**Pull request by AnnaxGrace**
Updated the muiTheme as well as the individual tags with disableTypography. With research, it looks like this the most recommended way, since dialogTitle (FOR SOME REASON) automatically creates an h2 tag with h6 variant stylings, so overrides and global do not work.

### CODE REVIEW

File: src/theme/muiTheme.js [hunk 1]
```diff
@@ -36,418 +36,422 @@ export const statusRed = "#FF6766";
...
+        height: "48px",
+      },
+    },
+    //Must use disable typography in the tag on your component
+    MuiDialogTitle: {
+      root: {
+        fontSize: "16px",
+        fontWeight: "500",
+        fontFamily: "Roboto",
+        lineHeight: "16px",
```
Comment (dhuangg) [inline src/theme/muiTheme.js:379]: since we have an h4 const at the top, can we do `root: { ...h4 }` instead?

**Review by dhuangg — CHANGES_REQUESTED** [review]
ahh ok so that was what I was missing. I ended up using disableTypography when I was working on this but I had hopes you didn't have to do that for everything for the theme styling to be applied.

File: src/theme/muiTheme.js (hunk 1 above)
Comment (AnnaxGrace) [reply src/theme/muiTheme.js]: Yes! I had tried to do it like that before and it didn't work... I didn't try using the ... spreader though! It's been updated :)

**Review by dhuangg — APPROVED** [review]
10,000 points to youuuu

**Pull request merged**